capstone/pom.xml

```diff
@@ -39,6 +39,18 @@
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-tools-sdk</artifactId>
+      <version>1.9.64</version>
+      <scope>test</scope>
```

Comment:
I believe this causes a double dependency warning when you deploy, do you need these to be two separate things? (appengine-testing and appengine-tools-sdk)

Reply:
They are two separate dependencies, appengine-testing is needed for data store testing

Reply:
I think you need to give them different groupIds then because its causing warnings when you build.

Reply:
That wasn't the problem, there was another dependency that was duplicated